git-open

```diff
@@ -49,6 +52,11 @@ IFS='=' read -ra suffix_flag <<< "$suffix_flag"
 function join_by { local IFS="$1"; shift; echo "$*"; }
 suffix=$(join_by "=" "${suffix_flag[@]:1}")
 
+# parse file from file=value
+IFS='=' read -ra file_flag <<< "$file_flag"
+function join_by { local IFS="$1"; shift; echo "$*"; }
```

Comment:
Why are you redefining this?

Reply:
The join_by function definition was cut and pasted, and could probably be removed.